recipes/seqan3/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
I think we could also do the package/install with cmake, but it will require more reading how to do it. For now let's do it like this.

https://cmake.org/cmake/help/v3.15/command/install.html